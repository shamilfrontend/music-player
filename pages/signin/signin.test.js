import { shallowMount } from '@vue/test-utils'

import Component from './index'

describe('SignInPage', () => {
  let instance

  beforeEach(() => {
    instance = shallowMount(Component)
  })

  test('should match snapshot', () => {
    expect(instance.html()).toMatchSnapshot()
  })
})
