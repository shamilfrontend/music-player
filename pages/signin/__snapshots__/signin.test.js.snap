// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInPage should match snapshot 1`] = `
<div>
  <h1>SignInPage</h1>
</div>
`;
